.github/workflows/static-site.yml

```diff
@@ -59,8 +56,22 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Deploy
-        run: yarn deploy:ci
-        if: github.event.pull_request.merged == true
+      - name: Deploy to preview channel
+        if: ${{ github.event_name != 'pull_request' }}
```

Comment:
Why this is a `!=` instead of a `==`?

Reply:
Good catch. I was testing the production step and reversed the operators of the last two steps. I just forgot to change this one back.

I've amended a commit for this and force-pushed the change.